package/cloudshell/iac/terraform/models/shell_helper.py

```diff
@@ -16,3 +17,9 @@ def __init__(self, api: CloudShellAPISession, res_id: str, tf_service: any, logg
         self.logger = logger
         self.sandbox_messages = sandbox_messages
         self.live_status_updater = live_status_updater
+
+        self.attr_handler = ServiceAttrHandler(
+            self.api,
+            self.sandbox_id,
+            self.tf_service
+        )
```

Comment:
its an anti-pattern to initialize services here. It should be provided as dependency injection.